src/components/data/DataNavigation.js

```diff
@@ -359,36 +360,48 @@ function DataNavigation(props) {
-                const home = respRoots.find(
-                    (root) => root.label === userProfile.id
-                );
-                home.icon = <HomeIcon />;
-                const sharedWithMe = respRoots.find(
-                    (root) => root.label === constants.SHARED_WITH_ME
-                );
-                setSharedWithMePath(sharedWithMe.path);
-                sharedWithMe.icon = <FolderSharedIcon />;
-                const communityData = respRoots.find(
-                    (root) => root.label === constants.COMMUNITY_DATA
-                );
-                setCommunityDataPath(communityData.path);
-                communityData.icon = <GroupIcon />;
-                const trash = respRoots.find(
-                    (root) => root.label === constants.TRASH
-                );
-                trash.icon = <DeleteIcon />;
+    const { error } = useQuery({
+        queryKey: ["dataFileSystemRoots", userProfile],
+        queryFn: getFilesystemRoots,
+        config: {
+            onSuccess: (respData) => {
+                if (respData && userProfile) {
+                    const respRoots = respData.roots;
+                    const home = respRoots.find(
+                        (root) => root.label === userProfile.id
+                    );
+                    home.icon = <HomeIcon />;
+                    const sharedWithMe = respRoots.find(
+                        (root) => root.label === constants.SHARED_WITH_ME
+                    );
+                    setSharedWithMePath(sharedWithMe.path);
+                    sharedWithMe.icon = <FolderSharedIcon />;
+                    const communityData = respRoots.find(
+                        (root) => root.label === constants.COMMUNITY_DATA
+                    );
+                    setCommunityDataPath(communityData.path);
+                    communityData.icon = <GroupIcon />;
+                    const trash = respRoots.find(
+                        (root) => root.label === constants.TRASH
+                    );
+                    trash.icon = <DeleteIcon />;
 
-                const basePaths = respData["base-paths"];
-                setUserHomePath(basePaths["user_home_path"]);
-                setUserTrashPath(basePaths["user_trash_path"]);
-                setDataRoots([home, sharedWithMe, communityData, trash]);
-            }
-        });
-    }, [userProfile]);
+                    const basePaths = respData["base-paths"];
+                    setUserHomePath(basePaths["user_home_path"]);
+                    setUserTrashPath(basePaths["user_trash_path"]);
+                    setDataRoots([home, sharedWithMe, communityData, trash]);
+                }
+            },
+            onError: () => {
+                //temporary workaround -> should still display community data
+                announce({
+                    text: formatMessage(intl, "dataSignIn"),
+                    variant: "info",
+                    vertical: "bottom",
```

Comment:
Yeah, the only reason I displayed this one on the bottom was because there was already so much blue on the top, I felt it was getting lost.  I can switch it for now.

Reply:
I need to adjust to theme color for `INFO` to use `info` intention instead of `primary`.